infra/cmake/packages/FlatBuffersConfig.cmake

```
@@ -26,7 +26,7 @@ function(_FlatBuffers_build)
                       INSTALL_DIR ${EXT_OVERLAY_DIR}
                       BUILD_FLAGS ${ADDITIONAL_CXX_FLAGS}
                       IDENTIFIER  "1.10-fix2"
```

```suggestion
                      IDENTIFIER  "1.10-fix3"
```

Please update identifier.